src/commands/run-tests.yml

```
@@ -36,6 +36,21 @@ parameters:
       with the specified tag.
     type: string
     default: ''
+  model-coverage-cobertura:
+    description: >
+      Path to write model coverage report in Cobertura XML format.
+    type: string
+    default: ''
+  test-results-simulink-test:
+    description: >
+      Path to write Simulink Test results in MLDATX format for Simulink Test test files.
```

This is the language we use in Azure.
```suggestion
      Path to export Simulink Test Manager results in MLDATX format.
```

---

Done.